src/sentry/integrations/bitbucket_server/client.py

```diff
@@ -144,6 +145,19 @@ def get_commits(self, project, repo, fromHash, toHash):
             params={"since": fromHash, "until": toHash, "merges": "exclude"},
         )
 
+    def get_last_commits(self, project, repo, limit=10):
```

Comment:
@pierredup How did you come up with the default number of `10` commits?

Reply:
This is based on the Bitbucket integration that also only used 10 commits

https://github.com/getsentry/sentry/blob/953683e2e0/src/sentry/integrations/bitbucket/repository.py#L116